### ISSUE TITLE
Möglich Bewegung an den Schläuchen kann zu Abnutzung führen / Potential movement on hoses could lead to wear

### ISSUE THREAD

**Issue by FelixThe** [DIY-Beatmungsgerat/diy-beatmungsgeraet#4]

**Comment by mattheiss2020** [comment]
It looks like a nice solution, but I just remember my physics class at school and I think that we would need quite a strong motor to be able to squeeze the Ambu bag? Can anybody calculate that?

**Comment by FelixThe** [comment]
Yes, you are right! It would need higher forces to squeeze the bag. I will think about a different solution for squeezing the bag from two sides.
